tests/Evenement/Tests/EventEmitterTest.php

```diff
@@ -85,6 +85,32 @@ public function testOnceWithArguments()
         $this->assertSame(array('a', 'b'), $capturedArgs);
     }
 
+    public function testOnceBefore()
+    {
+        $listenerCalled = 0;
```

Comment:
Like the idea, but could you change it into an array and test for it's contents to be `[1, 2, 3]` so that we know the full order?